internal/bminventory/inventory.go

```diff
@@ -4670,6 +4670,9 @@ func (b *bareMetalInventory) V2RegisterHost(ctx context.Context, params installe
 	log := logutil.FromContext(ctx, b.log)
 	log.Infof("Register host: %+v", params)
 
+	if params.NewHostParams.HostID == nil {
```

Comment:
isn't that validated on the swagger level? https://github.com/openshift/assisted-service/blob/96f0f2434fbadebf15b7edb697ec035488c3c872/swagger.yaml#L3820-L3823

Reply:
OK

Reply:
Removed